org.opensurge2d.OpenSurge.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: org.opensurge2d.OpenSurge
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+default-branch: stable
+sdk: org.freedesktop.Sdk
+command: opensurge
+rename-icon: opensurge
+rename-desktop-file: opensurge.desktop
+rename-appdata-file: opensurge.appdata.xml
+
+finish-args:
+  - --socket=x11
```

Comment:
Does it work with wayland?

Reply:
Allegro 5.2.5 does not work with Wayland afaik.

Reply:
Allegro 5.2.5 works with X.